model/src/main/kotlin/config/PostgresStorageConfiguration.kt

```diff
@@ -44,5 +44,34 @@ data class PostgresStorageConfiguration(
      * The password to use for authentication.
      */
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
-    val password: String = ""
+    val password: String = "",
+
+    /**
+     * The SSL mode to use, one of disable, allow, prefer, require, verify-ca or verify-full.
```

Comment:
Nit: How about double-quoting the mode names here to make them stand out in the sentence?

Reply:
Done.